sdstore/sdstore.go

```diff
@@ -297,7 +297,14 @@ func (s *sdStore) get(url *url.URL, toExtract bool) ([]byte, error) {
 		defer file.Close()
 	}
 
-	req, err := http.NewRequest("GET", url.String(), nil)
+	var urlString string
+	if toExtract == true {
```

Comment:
Is the `/` trimmed before this? User might pass in `/tmp/testcache/` instead of `/tmp/testcache`

Reply:
Nvm I see we have it inside `getFilePath`